home-application/src/main/java/com/softserveinc/ita/homeproject/config/DataLoader.java

```diff
@@ -0,0 +1,61 @@
+package com.softserveinc.ita.homeproject.config;
+
+import com.softserveinc.ita.homeproject.model.Permission;
+import com.softserveinc.ita.homeproject.model.Role;
+import com.softserveinc.ita.homeproject.model.User;
+import com.softserveinc.ita.homeproject.repository.PermissionRepository;
+import com.softserveinc.ita.homeproject.repository.RoleRepository;
+import com.softserveinc.ita.homeproject.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+@Profile("dev")
+public class DataLoader implements ApplicationRunner {
+
+    private final UserRepository userRepository;
+    private final RoleRepository roleRepository;
+    private final PermissionRepository permissionRepository;
+    private final PasswordEncoder passwordEncoder;
+
+    @Autowired
+    public DataLoader(UserRepository userRepository, RoleRepository roleRepository, PermissionRepository permissionRepository, PasswordEncoder passwordEncoder) {
+        this.userRepository = userRepository;
+        this.roleRepository = roleRepository;
+        this.permissionRepository = permissionRepository;
+        this.passwordEncoder = passwordEncoder;
+    }
+
+    public void run(ApplicationArguments args) {
```

Comment:
Is this method is overridden from ApplicationRunner? Please mark with appropriate annotation.